db/migrate/20221214150508_add_representer_version_to_tracks.rb

```diff
@@ -0,0 +1,11 @@
+class AddRepresenterVersionToTracks < ActiveRecord::Migration[7.0]
+  def change
+    unless Rails.env.production?
+      add_column :tracks, :representer_version, :smallint, null: true
+
+      ActiveRecord::Base.transaction(isolation: Exercism::READ_COMMITTED) do
+        Track.where(has_representer: true).update_all(representer_version: 1)
+      end
```

Comment:
```suggestion
      add_column :tracks, :representer_version, :smallint, null: false, default: 1
```

I ran this in production 😱 😉